youtube_dl/extractor/bandcamp.py

```diff
@@ -248,3 +261,62 @@ def _real_extract(self, url):
             'title': title,
             'entries': entries,
         }
+
+
+class BandcampWeeklyIE(InfoExtractor):
+    IE_NAME = 'Bandcamp:bandcamp_weekly'
+    _VALID_URL = r'https?://(?:www\.)?bandcamp\.com/\?(?:\w+=\w+&)*show=(?P<id>\d+)(?:&\w+=\w+)*#?'
```

Comment:
`(?:&\w+=\w+)*#?` is noop. Does not match `https://bandcamp.com/?blah@&show=224`.